### DISCUSSION TITLE
Profile feedback discussion

### DISCUSSION THREAD

**Discussion by hiringaitech** [community/community#66852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello Gities, I want to give a feed back how someone can improve the profile which could be anything like ReadMe, a GitHub website and it could be the open source contribution in  any domain.
What's your thoughts please share.

[Click for My Profile](https://github.com/hiringaitech/hiringaitech)

**Comment by Aliktk** [comment]
After going through your profile I see many things which are not yet completed like no proper ReadMe, no profile picture, no descriptions, no other details.
Please try to add, or you can get help from my profile as well here you can find me. [Ali Nawaz](https://github.com/Aliktk).

If this helps you please give this as the answer.
Thanks

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
